File: src/cpp/classes.cpp
#include <iostream>
#include <string>

using namespace std;

class MontyPython
{
  public:
    string FirstName;
    string LastName;

    string fullName()
    {
        return FirstName + " " + LastName;
    }

    MontyPython(string firstName, string lastName)
    {
        FirstName = firstName;
        LastName = lastName;
    }
};

int
main()
{
    MontyPython montyPython = MontyPython("John", "Cleese");
    cout << "Monty Python is " << montyPython.fullName() << "." << endl;

    montyPython.FirstName = "Eric";
    montyPython.LastName = "Idle";
    cout << "Now, Monty Python is " << montyPython.fullName() << ".";

    return 0;
}

File: src/php/classes.php
<?php
class MontyPython{
    public $FirstName;
    public $LastName;

    function __construct($firstName, $lastName){
        $this->FirstName = $firstName;
        $this->LastName = $lastName;
    }

    function FullName(){
        return $this->FirstName . " " . $this->LastName;
    }
}

$montyPython = new MontyPython("John", "Cleese");
echo "Monty Python is " . $montyPython->FullName() . ".\n";

$montyPython->FirstName = "Eric";
$montyPython->LastName = "Idle";
echo "Now, Monty Python is " . $montyPython->FullName() . ".";
?>